extensions/wikidata/module/styles/dialogs/wikibase-dialog.less

```diff
@@ -65,11 +65,15 @@
   color: gray;
 }
 
-.wikibase-list li a {
+.wikibase-list li a,span {
```

Comment:
@elebitzero it looks like there is a small error here: this CSS selector seems to be understood as "`.wikibase-list li a` OR `span`" (which is probably not what you intended). This means that the style instructions in it are applied to all spans in OpenRefine (even outside of this dialog). This results in unusually wide margins in quite a few places, for instance here:
![image](https://user-images.githubusercontent.com/309908/180461044-0b92937d-4712-44a1-9f82-52344d5a786c.png)
Both Chromium and Firefox seem to agree on the interpretation of this CSS selector.